server/services/stats/side/stats_side.go

```diff
@@ -28,19 +29,31 @@ type Config struct {
 	Enabled              bool              `yaml:"Enabled"`
 	Schedule             string            `yaml:"Schedule"`
 	NewRelicApiKey       string            `yaml:"NewRelicApiKey"`
+	TeamCityAddress      string            `yaml:"TeamCityAddress"`
```

Comment:
I didn't want to spend much time, customizing third-party library for test task. I'm still don't